### PR TITLE
Update development docs

### PR DESCRIPTION
**Pull request by Cjkjvfnby**
Here are some useful tips for AI development.

@Grummel7 , could you take a look?

### CODE REVIEW

File: default/python/README.md [hunk 1]
```diff
@@ -104,7 +104,24 @@ Black can be run on the top level directory:
 black .
 ```
 
-
+## Debug via chat
+This is an REPL to AI.
```
Comment (geoffthemedio) [inline default/python/README.md]: I had to google `REPL` so maybe write it out?

**Comment by geoffthemedio** [comment]
@Grummel7 ?

File: default/python/README.md [hunk 2]
```diff
@@ -110,3 +82,75 @@ Hook will track config file and update himself in case of changes.
...
+Settings for flake8 located in the `tox.ini` across the project.
+
+Flake should be run from directory where `tox.ini` located:
+```sh
+cd default/python
+flake8
+```
+
+We use [black](https://pypi.org/project/black/) for code formatting.
+Settings for black are located in `pyproject.toml`.
```
Comment (Grummel7) [inline default/python/README.md:107]: Perhaps isort and pytest could be mentioned as well. I constantly learn about new tests when one of my PRs fail the github hooks.

File: default/python/README.md [hunk 3]
```diff
@@ -23,64 +23,37 @@ editing options.py and universe_tables.py, both of which have more information
...
+# Development
+
+## Setup environment
+### Run game with AI files from the repository folder
+- Run game
+- Open Options
+- Scroll left to Others
+- Click generate persistent config
+- Check `Congig and log files path` value
+- Open directory with config
```
Comment (Grummel7) [inline default/python/README.md:37]: Typo: Congig

"Run game with AI files from the repository folder": So far I wasn't aware you can use anything other than the repository folder. When compile the game, this is the default.

File: default/python/README.md [hunk 4]
```diff
@@ -23,64 +23,37 @@ editing options.py and universe_tables.py, both of which have more information
...
+- Edit `persistent_config.xml`
+- ```xml
+  <?xml version="1.0"?>
+  <XMLDoc>
+      <resource>
+      <path>...repo_path\default</path>
+      <stringtable>
```
Comment (Grummel7) [inline default/python/README.md:44]: Indent of <ressource> is wrong.

The generated persistent_config.xml contains a lot, but no <resource> tag. Bug in the generation code?
I guess this does not mean to remove all other values.

File: default/python/README.md (hunk 3 above)
Comment (Cjkjvfnby) [reply default/python/README.md:37]: I don't compile on my machine (windows), so I use weekly builds.

File: default/python/README.md (hunk 4 above)
Comment (Cjkjvfnby) [reply default/python/README.md:44]: It merged with an override to the config.  Config is overridden on game install.

File: default/python/README.md (hunk 2 above)
Comment (Cjkjvfnby) [reply default/python/README.md:107]: Added

**Pull request merged**